### DISCUSSION TITLE
[Feature Request] Charts

### DISCUSSION THREAD

**Discussion by jeffscottward** [cschroeter/park-ui#398] [General]
https://ui.shadcn.com/charts

You know the drill.

**Comment by piyanggoon** [comment]
+1

**Comment by phlzZz** [comment]
On it. 🧑🏽‍🎨

**Comment by cschroeter** [comment]
@phlzZz should not be hard to beat ;)

**Comment by jeffscottward** [comment]
Has this been thought about? Started on? What's the status?

**Reply by jeffscottward** [reply]
@phlzZz

**Reply by phlzZz** [reply]
Hey! We are going to work on them but we haven't had time yet. Both Chris and I are on vacation right now.
